bundles/fluent-en_GB/extensions/atrium-fluent-en_GB-smoke-test-jdk8/build.gradle

```diff
@@ -6,5 +6,5 @@ targetCompatibility = JavaVersion.current()
 dependencies {
     //I don't see how to set up compileTestKotlin with --patch-module, so we have put the module-info.java directly in src/test/kotlin instead
     testImplementation prefixedProject('fluent-en_GB-jvm')
-    testImplementation prefixedProject('api-fluent-en_GB-jdk8-jvm')
+    // testImplementation prefixedProject('api-fluent-en_GB-jdk8-jvm')
```

Comment:
I suggest we remove atrium-fluent-en_GB-smoke-test-jdk8 entirely

Reply:
delete the whole folder?

Reply:
I would delete the whole folder. Check in settings.gradle in the root that it is excluded as well.
What would be nice, move:
```
    test("see if `Path.existsNot` can be used") {
        assert(Paths.get("nonExisting")).existsNot()
    }
``` 
from bundles/fluent-en_GB/extensions/atrium-fluent-en_GB-smoke-test-jdk8/src/test/kotlin/custom/SmokeSpec.kt to bundles/fluent-en_GB/atrium-fluent-en_GB-jvm/src/test/kotlin/custom/SmokeSpec.kt